packages/runtime/src/runtime.ts

```diff
@@ -608,15 +628,28 @@ export class Runtime {
    * @param args : external arguments to smart contract
    */
   executeTx (txnParams: ExecParams | ExecParams[]): void {
-    const [tx, gtxs] = this.createTxnContext(txnParams); // get current txn and txn group (as encoded obj)
+    const txnParameters = Array.isArray(txnParams) ? txnParams : [txnParams];
+    for (const txn of txnParameters) {
+      switch (txn.type) {
+        case TransactionType.DeployASA: {
+          txn.asaDef = this.loadedAssetsDefs[txn.asaName];
+          break;
+        }
+        case TransactionType.DeploySSC: {
+          txn.approvalProg = new Uint8Array(32); // mock approval program
+          txn.clearProg = new Uint8Array(32); // mock clear program
+          break;
```

Comment:
could you please explain why we set mock values, instead of `approval`, `clear` programs while deploying ssc? Is it because it's not the compiled teal code?

Reply:
yes, and we don't need it while testing